apps/src/sites/studio/pages/scripts/lesson_extras.js

```diff
@@ -22,8 +22,8 @@ config.bonusLevels = config.bonusLevels.map(bonus => {
 ReactDOM.render(
   <Provider store={store}>
     <LessonExtras
-      lessonNumber={config.stageNumber}
-      nextLessonNumber={config.nextStageNumber}
+      lessonNumber={config.lessonNumber}
+      nextLessonNumber={config.nextLessonNumber}
```

Comment:
looks like you would need to update this codepath also:
* https://github.com/code-dot-org/code-dot-org/blob/842dc7e4d76a70531c44a0f372e39b1c1e5f5f08/dashboard/app/controllers/script_levels_controller.rb#L272-L274
* https://github.com/code-dot-org/code-dot-org/blob/503d587324b19cfe9993c316e0166de1db9f2c32/dashboard/app/views/scripts/lesson_extras.html.haml#L7

Reply:
Thanks for your thoughtful review. I think this is fixed now